### PR TITLE
add chips and update giant-squid

### PR DESCRIPTION
**Pull request by d3v-null**

### CODE REVIEW

File: repo/packages/chips/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+class Chips(MakefilePackage):
+
+    homepage = "https://github.com/cathryntrott/chips2024"
+    # git = "https://github.com/cathryntrott/chips2024.git"
+    git = "https://github.com/d3v-null/chips2024.git"
+
+    maintainers('d3v-null')
+
+    version("main", branch="main")
```
Comment (dipietrantonio) [inline repo/packages/chips/package.py:11]: In addition to the "main" branch you could specify a particular commit, just to have the option of a reproducible installation (main branch can be updated).

File: repo/packages/chips/package.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    depends_on('cfitsio')
+    depends_on('openblas')
+    depends_on('fftw')
+    depends_on('pal')
+    depends_on('gsl')
+    depends_on("gmake", type="build")
+
+    def install(self, spec, prefix):
```
Comment (dipietrantonio) [inline repo/packages/chips/package.py:20]: Is this needed? I would think MakeFile package would handle that

**Comment by dipietrantonio** [comment]
I have tested the recipe. It installs and the software runs fine. Please @d3v-null address my comment on adding specific versions for reproducible installations and executions.

**Review by dipietrantonio — APPROVED** [review]

**Comment by pelahi** [comment]
I will also approve if we can add some versioning.

**Review by pelahi — APPROVED** [review]

**Pull request merged**